dandi/files/bases.py

```diff
@@ -19,7 +19,7 @@
 from dandischema.models import Dandiset as DandisetMeta
 from dandischema.models import get_schema_version
 from etelemetry import get_project
-from nwbinspector import Importance, inspect_nwb, load_config
+from nwbinspector import Importance, inspect_nwbfile, load_config
```

Comment:
Do we need to boost minimal version for nwbinspector ?

Reply:
I don't know what version `inspect_nwbfile()` was added in.  CC @CodyCBakerPhD

Reply:
It was added in the latest version, so yes I would bump minimal.

Reply:
latest = `0.4.28`

Keep in mind `etelemetry` complains (just a warning I think) if running validation when not using the latest release too, but upgrading the requirement is never a bad idea either